exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,3 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
```

Comment:
You'll need to do some calculations here to get the percentages.

E.g. like in school `(numberOfStudents / (numberOfStudents + numberOfMentors)) * 100`.